src/views/catalog/CreateFromInstanceTypes/CreateFromInstanceType.scss

```diff
@@ -24,6 +24,16 @@
   }
   &__header {
     font-size: var(--pf-global--FontSize--xl);
+
+    .help-icon__cursor {
+      color: var(--pf-global--Color--dark-100);
+      font-size: var(--pf-global--FontSize--sm);
+      vertical-align: 0rem !important;
+    }
```

Comment:
why do we want to change the cursor?

Reply:
We don't change the cursor even when it looks like that. We change size and color of "?" icon with this change, to achieve the expected look as in the design doc.